sys-power/brightnessctl/brightnessctl-0.3.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A program to read and control device brightness"
+HOMEPAGE="https://github.com/Hummer12007/brightnessctl"
+
+SRC_URI="https://github.com/Hummer12007/${PN}/archive/${PV}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="+suid"
+
+src_install() {
+	emake DESTDIR="${D}" INSTALL_UDEV_RULES="$(usex suid 0 1)" install
+
+	#doman ${PN}.1
```

Comment:
This manual has been commit to master but no release has been made it. Asked for one.

Reply:
I have removed this commented out `doman` since a release is not available yet. I can add it later on a future release.